src/aiq/agent/react_agent/agent.py

```diff
@@ -195,14 +195,11 @@ async def conditional_edge(self, state: ReActGraphState):
             logger.debug("Starting the ReAct Conditional Edge")
             if len(state.messages) > 1:
                 # the ReAct Agent has finished executing, the last agent output was AgentFinish
-                if self.detailed_logs:
-                    logger.debug("Final answer:\n%s", state.messages[-1].content)
+                logger.debug("Final answer:\n%s", state.messages[-1].content)
                 return AgentDecision.END
             # else the agent wants to call a tool
             agent_output = state.agent_scratchpad[-1]
-            if self.detailed_logs:
-                logger.debug("the agent wants to call: %s with input: %s", agent_output.tool, agent_output.tool_input)
-            logger.debug('Agent is calling a tool')
+            logger.debug("the agent wants to call: %s with input: %s", agent_output.tool, agent_output.tool_input)
```

Comment:
```suggestion
            logger.debug("The agent wants to call: %s with input: %s", agent_output.tool, agent_output.tool_input)
```

Make sure to check for correct punctuation as well.